src/test/end-to-end/helpers/caseHelper.js

```diff
@@ -129,7 +131,18 @@ async function printHearingLists(I, eventName, jurisdiction) {
 async function caseTransfer(I, eventName) {
     await I.chooseNextStep(eventName, 3);
     await I.wait(2);
-    await I.executeCaseTransfer();
+    switch (eventName){
+        case "Case Transfer (Eng/Wales)":
+            await I.executeCaseTransferEngWales();
+            break;
+        case "Case Transfer (Scotland)":
+            await I.executeCaseTransferScotland();
+            break;
+        case "Case Transfer to ECM":
+            await I.executeCaseTransferECM();
+            break;
+        default:
+    }
```

Comment:
Throw an error in case the control comes to the default.... block...

Reply:
Added throw new Error("Control arrived at default block");